Rocket.Chat/Managers/Model/SubscriptionManager.swift

```diff
@@ -353,10 +353,13 @@ extension SubscriptionManager {
         }
     }
 
-    static func sendTextMessage(_ message: Message, completion: @escaping MessageCompletion) {
+    static func sendEditTextMessage(_ message: Message, isEdit: Bool = false, completion: @escaping MessageCompletion) {
+
+        let methodName: String = isEdit ? "updateMessage" : "sendMessage"
+
         let request = [
             "msg": "method",
-            "method": "sendMessage",
+            "method": methodName,
```

Comment:
Let's have two separated methods here instead of changing the method name based on `isEdit`. OK?